### DISCUSSION TITLE
Release Cadence

### DISCUSSION THREAD

**Discussion by 243826** [diffplug/spotless#1885] [Q&A]
Is it defined?

**Comment by nedtwigg** [comment]
Nope. I usually click "ship it" only when the changelog gets big enough that it feels worth it to spam everybody's "renovate" bots.

**Reply by 243826** [reply]
Should there be a monthly cadence? Right now there is no release since September and I cant predict when it would be.

**Reply by nedtwigg** [reply]
If there is something in the unreleased backlog which you want to build against, you can [use JitPack](https://github.com/diffplug/spotless/blob/main/CONTRIBUTING.md#gradle---any-commit-in-a-public-github-repo-this-one-or-any-fork) to build against any commit that you want.

A monthly cadence is not unreasonable. I'll think about it a bit. If I haven't released already by December 1st, ping me again here and I'll cut a release.
